### PR TITLE
Adding Multiarch platforms

### PR DESCRIPTION
**Pull request by deepsm007**
Adding the architecture to support multiarch. 

/cc @deads2k

### CODE REVIEW

File: pkg/jobrunaggregator/jobtableprimer/job_typer.go [hunk 1]
```diff
@@ -15,6 +15,8 @@ func newJob(name string) *jobRowBuilder {
 	switch {
 	case strings.Contains(name, "gcp"):
 		platform = gcp
+	case strings.Contains(name, "arm64"):
```
Comment (deads2k) [inline pkg/jobrunaggregator/jobtableprimer/job_typer.go:18]: Are these actually platforms, not a new field type call architecture?

Comment (deepsm007) [reply pkg/jobrunaggregator/jobtableprimer/job_typer.go:18]: In theory there are architecture, thats where my original confusion came along as we want it to be part of separate field

**Pull request merged**